### ISSUE TITLE
An endpoint in the Rest API for deleting meshes

### ISSUE THREAD

**Issue by Frissj** [NVIDIAGameWorks/rtx-remix#673]
### What is the context of the feature/improvement?

I would like to add the ability to automatically delete the old mesh that had been imported to blender with my addon. This saves you time as you don’t have to hunt around for overlapping geometry.

### Please describe the feature/improvement?

Adapting the replace mesh endpoint to delete a mesh if the content is set to None for example.

### Version

2024.4.1

**Comment by automata-rtx** [comment]
Yes this would be great. I used his addon to replace a barrier prop in my project and it worked flawlessly. However the original mesh was still present and needed to be manually deleted. Would be a great QoL feature if the API supported such a feature so he could implement it into his Blender addon

**Comment by po-trottier** [comment]
Hey, just looping back, can this ticket be closed?
